docs: add extra information about Ubuntu Pro DOC-1843 (#6547)
spectrocloud/librarium: docs/docs-content/vertex/fips/fips-compliant-components.md

- Container Storage Interface (CSI)
  - vSphere CSI

:::info

> Please note that Ubuntu Pro subscription keys must be obtained independently or purchased through Spectro Cloud. Get in
> touch with your support representative to find out more.

Reviewer: ```suggestion
Ubuntu Pro subscription keys must be obtained independently or purchased through Spectro Cloud. Contact your support representative to learn more.
```
Unless we have to implement word-for-word what Joi said, I think something like the above works better. Cuts down on wordiness and is more direct.
